t/18-job-constants.t

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env perl -w
```

Comment:
Please delete the "-w" suffix if you agree with https://github.com/os-autoinst/openQA/pull/2794